org.kde.kget.json

```diff
@@ -0,0 +1,135 @@
+{
+  "id" : "org.kde.kget",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.15-21.08",
+  "sdk" : "org.kde.Sdk",
+  "command" : "kget",
+  "rename-icon" : "kget",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=xdg-download",
+    "--share=network"
+  ],
+  
+  "modules" : [
+    {
+      "name" : "boost",
+      "buildsystem" : "simple",
+      "sources" : [
+        {
+          "type" : "archive",
+          "url" : "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
```

Comment:
This is an "old" version. I think the current one is 1.78.

Reply:
do i need to update link for those too which are already in runtime? like gpgme etc.